Packs/Netskope/TestPlaybooks/playbook-NetskopeAPIv2_Test.yml

```
@@ -0,0 +1,864 @@
+id: Netskope
```

```suggestion
id: Netskope_V2_Test
```
change the name as well